src/client/components/pages/index.js

```diff
@@ -262,7 +235,7 @@ class IndexPage extends React.Component {
 IndexPage.displayName = 'IndexPage';
 IndexPage.propTypes = {
 	disableSignUp: PropTypes.bool,
-	recent: PropTypes.array.isRequired
+	results: PropTypes.array.isRequired
```

Comment:
Originally when I asked you to rename 'recent' to 'results', I only had in mind the revisions page, not the index page.
Semantically, it makes a lot more sense to have a 'recent' or 'recentRevisions‘ on the index page.
On the revisions page, it's obvious that we have 'results' of a query (last X revisions, page Y), but on the index page what are they the results of?

This will need changing in a few places, and I now it's an annoying little change, but I keep looking at it and it bugs me :p

Reply:
It's alright I'll change this :)